src/types/widget.ts

```diff
@@ -61,12 +61,18 @@ export type UiState = {
   range?: {
     [attribute: string]: string;
   };
-  starRating?: {
-    [attribute: string]: number;
-  };
   toggle?: {
     [attribute: string]: boolean;
   };
+  geoSearch?: {
+    /**
+     * The rectangular area in geo coordinates.
```

Comment:
The description is taken from the [`insideBoundingBox`](https://www.algolia.com/doc/api-reference/api-parameters/insideBoundingBox/?language=javascript) API ref.